specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/quota.json

```diff
@@ -0,0 +1,1226 @@
+      "x-ms-enum": {
+        "name": "ResourceQueryType",
+        "modelAsString": true
+      }
+    },
+    "ResourceQueryMethod": {
+      "description": "The resource query method. ",
+      "enum": [
+        "GET",
+        "POST"
+      ],
+      "readOnly": false,
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ResourceQueryMethod",
+        "modelAsString": true
+      }
+    },
+    "QuotaTemplatesDetails": {
+      "type": "object",
+      "description": "Quota templates details.",
+      "properties": {
+        "value": {
+          "description": "The quota templates information.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QuotaTemplateDetails"
+          }
+        }
+      }
+    },
+    "ServiceError": {
+      "description": "The api error details.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServiceErrorDetail"
+          },
+          "description": "The list of error details.",
+          "readOnly": true
+        }
+      }
+    },
+    "ServiceErrorDetail": {
+      "description": "The error details.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The error message."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdInParameters": {
+      "name": "subscriptionId",
+      "x-ms-parameter-location": "method",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Azure subscription id."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "API version."
+    },
+    "LocationIdInParameters": {
+      "name": "location",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Azure region.",
+      "x-ms-parameter-location": "method"
+    },
+    "ProviderIdInParameters": {
+      "name": "providerId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Azure resource provider id.",
+      "x-ms-parameter-location": "method"
+    },
+    "ResourceNameInParameters": {
+      "name": "resourceName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The resource name for a resource provider, such as SKU name for Microsoft.Compute, Sku or TotalLowPriorityCores for Microsoft.MachineLearningServices",
```

Comment:
>description [](http://example.com/codeflow?start=7&length=11)

Can you please clarify this description? I mean, if I have a VM called foo, the resource name is going to be foo right? Is this what resourceName is? Just want to make sure customers know what resourceName to use.

Reply:
Chatted offline, this is meant for SKUs